runtime/doc/api.txt

```diff
@@ -179,6 +179,7 @@ Nvim exposes its API metadata as a Dictionary with these items:
                           describing the return value and parameters.
 - ui_events               |UI| event signatures
 - ui_options              Supported |ui-option|s
+- ui_element_tags         String to bit number mapping for |ui-elementtype|
```

Comment:
what is "tags" doing for us in this name
```suggestion
- ui_elements         String to bit number mapping for |ui-elementtype|
```

Reply:
Tags was added because by itself a single tag is not necessarily an UI element by itself, for example `Left` is one of the possible tags.

But if you prefer it can be changed to `ui_elements`